notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -656,11 +641,11 @@
    "source": [
     "## Create Seed Dataset for SDG\n",
     "\n",
-    "This notebook combines the contents from the qna.yaml and the chunks from the source document to create a seed dataset for the synthetic data generation process.\n",
+    "This section combines the contents from the qna.yaml and the chunks from the source document to create a seed dataset for the synthetic data generation process.\n",
```

Comment:
I think this description needs a refactor because I don't quite understand what is happening here